app/controllers/admin/merchants_controller.rb

```diff
@@ -11,4 +11,11 @@ def disable
     redirect_to "/admin/merchants"
     flash[:notice] = "#{@merchant.name} is disabled"
   end
+
+  def enable
+    @merchant = Merchant.find(params[:merchant_id])
```

Comment:
Does this need to be an instance variable?

Reply:
Great question... let me check.

Reply:
It doesn't! I'll change that when I do US #41